eventhubs/spring-cloud-azure-stream-binder-eventhubs/eventhubs-binder/src/main/resources/application.yaml

```diff
@@ -0,0 +1,33 @@
+spring:
+  cloud:
+    azure:
+      eventhubs:
+        connection-string: ${AZURE_EVENTHUBS_CONNECTION_STRING}
+        processor:
+          checkpoint-store:
+            container-name: ${AZURE_STORAGE_CONTAINER_NAME}
+            account-name: ${AZURE_STORAGE_ACCOUNT_NAME}
+            account-key: ${AZURE_STORAGE_ACCOUNT_KEY}
+    stream:
+      function:
+        definition: consume;supply
+      bindings:
+        consume-in-0:
+          destination: ${AZURE_EVENTHUB_NAME}
+          group: ${AZURE_EVENTHUB_CONSUMER_GROUP}
+          consumer:
```

Comment:
Now that the default value is fault, why should we write it again in the default configuration? We should delete this because this will confuse users. Same for the readme.

Reply:
updated both place